src/frontend/js/modules/i18n/i18n.run.js

```diff
@@ -1,10 +1,23 @@
+'use strict';
+
 require('./i18n.service.js');
+require('./i18n.constants');
+
+angular.module('esn.i18n')
+  .run(setMomentLocale)
+  .run(settingLanguage);
```

Comment:
A question: Why moving `settingLanguage` run block to another place can fix this bug?

Reply:
Not only moving but also execute the run block. In the old place, the file is not loaded in SPAs, so the run block is not executed. Move it to i18n run block and SPAs require i18n module, then the block is executed.

Reply:
ok, I understand